Wikibase/Tests/Integration/LineLength80.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+// phpcs:property PHP_CodeSniffer\Standards\Generic\Sniffs\Files\LineLengthSniff::$absoluteLineLimit = 80
```

Comment:
This is a home-brew addition. I'm not sure if it's a good idea to copy the `phpcs:…` prefix. Do you have a suggestion?

Reply:
I am not that fond of this then, to be honest. What do you think about using what PHP Unit has to offer, instead of coming up with our own standard to encode test input?

Reply:
Not fond of it as well.
Reasons:
  * It is yet another language. It is not checked by PHP compiler and may easily contain a mistake which will be hard do spot.
  * It is in comment which is part of the file. If you add one, then all the line numbers in `*.php.expected` have to be increased by one. It will become very annoying when you developing new sniff. Or, in worst case (very unlikely but still), it might be even incompatible with the sniff itself.
  * It doesn't allow you to have tests with different values of the property.

Regarding above I would say to do it properly, we need to put quiet some effort and implement proper parsing, and probably move property logic into the sniffs, to make it possible to change property value for each piece of code so we can test sniff with different property values. I don't see any need to do that.